Automattic/wp-calypso
# Help Center: Use default values on case of error

This is a bit oppinionated but the hc should be available no matter what.
This PR makes things a bit more resilient. 
In case we fail to load a odie chat, the help center should continue working.

Related to p1733303607767299/1733233447.208929-slack-C02T4NVL4JJ

## Proposed Changes

* Add default values for messages and chat id

## Why are these changes being made?


* To ensure that users can reach support

## Testing Instructions



* Checkout this PR
* Force the odie assistant endpoint to error out. You can force `sources` to be string before iterating. fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Serfg%2Qncv%2Qcyhtvaf%2Sraqcbvagf%2Sbqvr%2Qnffvfgnag.cuc%3Se%3Q00nq0488%23482-og 
* Clicking on new conversation should open properly
